### PR TITLE
bufix: StatusText component

### PR DESCRIPTION
**Pull request by anantakumarghosh**
## Description

This PR fixes StatusText component
#313

## Checklist

- [x] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [x] My code follows the project's style guidelines and best practices.
- [x] I have updated the documentation if necessary.
- [x] I have added or updated relevant comments in my code.
- [x] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: package/components/dataDisplay/StatusText.js [hunk 1]
```diff
@@ -3,16 +3,44 @@ import React from "react";
...
-      {status || "UNKNOWN"}
+      {...props}
+      styleClasses={[getStatusTextColorClass(props?.status || ""), CoreClasses.TEXT.TEXT_WEIGHT_BOLD, CoreClasses.TEXT.TEXT_UPPERCASE]}>
+      {props?.status || "unknown"}
     </CoreTypographyCaption>
   );
 }
+
+StatusText.validProps = [
+  {
```
Comment (techoneel) [inline package/components/dataDisplay/StatusText.js:24]: missing core typography caption props in valid props please spread the valid props of CoreTypographyCaption

**Review by techoneel — APPROVED** [review]
please do the changes

File: package/components/dataDisplay/StatusText.js (hunk 1 above)
Comment (anantakumarghosh) [reply package/components/dataDisplay/StatusText.js:24]: Are we supporting ALL props of CoreTypographyCaption? What are the requirements of StatusText? Please write

Comment (anantakumarghosh) [reply package/components/dataDisplay/StatusText.js:24]: `status` shouldn't go directly

**Pull request merged**